e2e/cypress/test_module/cypress/support/step_definitions/dashboard.stepts.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress" />
+
+import { Given, When, Then } from "@badeball/cypress-cucumber-preprocessor";
+import { UserRole } from "@cypress-e2e/support";
+
+Given(
+  "I authenticate to the system under {string} role",
+  (role: UserRole) => {
+    cy.loginWithRole(role);
+  }
+);
```

Comment:
I believe we already have step in `common.steps.ts`

Reply:
I missed it. Thank you